key.go

```diff
@@ -31,6 +31,53 @@ const (
 		BackQuote
 	*/
 
+	KeyApostrophe   = "K_APOSTROPHE"
+	KeyComma        = "K_COMMA"
+	KeyMinus        = "K_MINUS"
+	KeyPeriod       = "K_PERIOD"
+	KeySlash        = "K_SLASH"
+	Key0            = "0"
```

Comment:
I wasn't sure if we needed these - would the API be better if folk used "0" directly, or should we define all these consts?

Reply:
These are not directly used right now, but could be used in the future to detect shortcut that are based on layout keys like copy/paste (KeyC, KeyV, ...)
Anyway we could remove not strictly needed constants :)